lib/utils/npm-packages-exists.js

```diff
@@ -1,6 +1,7 @@
 "use strict";
 const chalk = require("chalk");
-const npmExists = require("./npm-exists");
+const packageExists = require("./package-exists");
+const path = require("path");
```

Comment:
Path should be above packageExists

Reply:
maybe its work enabling absolute imports eslint check?